chapters/methodology.tex

```diff
@@ -14,3 +14,25 @@ \subsection{State Machine}
 \subsection{User Interface}
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
+\subsection{Model Training}
+
+As stated in the background section, soHappy uses a deep learning model to 
+provide information about a person smiling by utilizing TensorFlow.
+
+However, for TFLite to detect smiling, the model must be trained first. This
+model is not trained on the device, but will be trained in before and shipped
+with the android app.
+
+The soHappy smile detection model is based a convolutional neural network 
+(CNN). CNNs are useful for image analysis purpuses. A CNN utilizes multiple
+convolutional layers as well as pooling layers. A convolutional layer works by

```

Comment:
Since you briefly describe what a convolutional layer does, I would do the same for pooling layers.